docs/source/providers/local.rst

```diff
@@ -0,0 +1,36 @@
+Adding a Local Provider
+#######################
+
+This section describes the local providers that are used for development and
+testing purposes. Local providers give Koku the ability to access test data
+without requiring a dependency on an external service, such as AWS S3 or a
+pre-existing OpenShift cluster (OKD).
+
+Configuring an AWS Local Provider
+*********************************
+
+Local Provider with OpenShift
```

Comment:
Should probably be cleaned up to be AWS Local Provider.

Any other comments or update to make here @dccurtis